sim_module_test_sketch/sim_module_test_sketch.ino

```diff
@@ -1,38 +1,41 @@
 #include <SoftwareSerial.h>
+SoftwareSerial mySerial(3, 2); //SIM800L Tx & Rx is connected to Arduino D3 & D2
 
-//Create software serial object to communicate with SIM800L
-SoftwareSerial mySerial(3, 2);  //SIM800L Tx & Rx is connected to Arduino D3 & D2
-
-void setup() {
-  //Begin serial communication with Arduino and Arduino IDE (Serial Monitor)
+void setup()
+{
   Serial.begin(9600);
-
-  //Begin serial communication with Arduino and SIM800L
+  
   mySerial.begin(9600);
 
-  Serial.println("Initializing...");
+  Serial.println("Initializing..."); 
```

Comment:
Here you just added redundant space. We do not need to have such unnecessary changes

Reply:
Fixed.